<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LidWatcherSettings</class>
 <widget class="QWidget" name="LidWatcherSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>289</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox QGroupBox {
border: none;
padding-top: 30px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="lidWatcherSettingsGroupBox">
     <property name="title">
      <string>Enable &amp;Lid Watcher</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="title">
         <string>Action when lid is closed</string>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <property name="labelAlignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="onBatteryActionLabel">
           <property name="text">
            <string>On Battery</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="onBatteryActionComboBox">
           <item>
            <property name="text">
             <string>Nothing</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Suspend</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Hibernate</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Shut down</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="onAcActionLabel">
           <property name="text">
            <string>On AC</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="onAcActionComboBox"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="extMonGroupBox">
        <property name="title">
         <string>When &amp;using external monitor</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <property name="labelAlignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="extMonOnBatteryActionLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>On Battery</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="extMonOnBatteryActionComboBox"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="extMonOnAcActionLabel">
           <property name="text">
            <string>On AC</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="extMonOnAcActionComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
